styles/styles.css

```diff
@@ -0,0 +1,268 @@
+  background-image: linear-gradient(90deg, var(--light-rainy-sky), var(--cloud-gray));
+}
+.rainy h1 {
+  background-image: linear-gradient(var(--light-gray-blue), var(--dark-gray));
+}
+.sunny h1 {
+  background-image: linear-gradient(var(--blue-sky), var(--bright-sun));
+}
+.windy h1 {
+  background-image: linear-gradient(var(--pale-sun), var(--blue-sky));
+}
+.foggy h1 {
+  background-image: linear-gradient(var(--fog-gray), var(--mid-fog-gray));
+}
+.snowy h1 {
+  background-image: linear-gradient(var(--pale-sky), white);
+}
+a:link {
+  color: #1c2958;
+}
+a:visited {
+  color: #161f43;
+}
+a:hover {
+  color: #3857b7;
+}
+a:active {
+  color: #0042ff;
+}
+sup {
+  white-space: nowrap;
+}
+select {
+  margin-top:.25rem;
+  font-size: 1.25rem;
+  display: block;
+  width:100%;
+}
+#container {
+  height:100%;
+  padding: 1vh 2.5vw;
+}
+#container.cloudy {
+  background-image: linear-gradient(var(--pale-sky-tr),var(--cloud-gray-tr));
+}
+#container.rainy {
+  background-image: linear-gradient(var(--light-rainy-sky-tr),var(--rain-cloud-blue-tr));
+}
+#container.sunny {
+  background-image: radial-gradient(circle at 5% 5%, var(--bright-sun-tr),var(--blue-sky-tr),var(--mid-blue-tr));
+}
+#container.windy {
+  background-image: radial-gradient(circle at 15% 5%, var(--pale-sun-tr),var(--deep-blue-sky-tr),var(--bright-sky-tr));
+}
+#container.foggy {
+  background-image: linear-gradient(var(--fog-gray-tr),var(--mid-fog-gray-tr));
+}
+#container.snowy {
+  background-image: linear-gradient(var(--snowy-sky-tr),var(--darker-gray-tr));
+}
+.page-header {
+  text-align: center;
+}
+#controls {
+  padding: 0.125rem;
+}
+#controls {
+  background-image: url(../assets/seasons.jpg);
+  /* 5%, 32.5%, 62.5%, 92.5 */
+}
+#controls.spring {
+  background-position: 0 5%;
+}
+#controls.summer {
+  background-position:0 32.5%;
+}
+#controls.fall {
+  background-position:0 62.5%;
+}
+#controls.winter {
+  background-position:0 92.5%;
+}
+.control,
+.daily-weather {
+  background-color: var(--silver-tr);
+  border: 5px solid var(--super-light-gray-tr);
+
+  border-radius: 5px;
+  margin: 0 0.25rem;
+  padding: 0.25rem;
+}
+.control input {
+  width: 92%;
+}
+.daily-weather {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+#unit-toggle {
+  font-size: 0.5em;
+  font-weight: normal;
+}
+#unit-toggle a {
+  color: var(--mid-gray-tr);
+}
+#unit-toggle .unit.selected {
+  color: inherit;
+  font-weight: bold;
+}
+.temperature {
+  font-size: 2em;
+  font-weight: bold;
+}
+.temperature.today {
+  font-size: 3em;
+}
+#temp-today {
+  display: inline-block;
+  width:6rem;
+}
+.temp-toggle {
+  width: 50%;
+}
+.long-word {
+  font-size: 8vw;
+  word-break: break-word;
+}
+#garden-sky,
+#garden-plot {
+  font-size: 2.5em;
+  text-shadow: 0 4px 8px var(--darker-gray-tr);
+  padding: 1rem 5%;
+  margin-top: 0.125rem;
+}
+#garden-plot {
+  margin-top: 0;
+  box-shadow: 0 0 10px var(--super-light-gray-tr);
+}
+@media (min-width: 274px) {
```

Comment:
yayy media queries. Here is one of my favorite resources on media queries https://css-tricks.com/a-complete-guide-to-css-media-queries/ and another one on flexbox + media queries https://thoughtbot.com/blog/flexboxes-media-queries-awesome-layouts